src/data/Profile.json

```diff
@@ -313,5 +313,34 @@
       "Twitter": "https://twitter.com/liza_bhutani",
       "LinkedIn": "https://www.linkedin.com/in/liza-bhutani"
     }
-  }
+  },
+	{
+		"name": "Yash Sehgal",
+		"location": "Indore, India",
+		"bio": "Software Engineer. Design with code",
+		"avatar": "https://github.com/yashsehgal.png",
+		"portfolio": "https://yashsehgal.com",
+		"skills": [
+			"Javascript",
+			"Typescript",
+			"React",
+			"NextJS",
+			"Astro",
+			"Git/GitHub",
+			"Linux",
+			"APIs",
+			"UI Design",
+			"Figma",
+			"CSS/SCSS",
+			"TailwindCSS",
+			"AntDesign",
+			"ChakraUI",
+			"RadixUI"
+		],
```

Comment:
I appreciate your skills but currently we can only render up to 7 skills. So please make changes and also update your branch with latest commit.

Reply:
I see this issue on the frontend. Rather limiting skills to 7 only, we can add horizontal scroll to avoid the breaking width on the page. What do you think about this?

If you want I can raise an issue and PR to fix this.

Reply:
Yeah, feel free to open a PR.

Reply:
A fix like this.

https://user-images.githubusercontent.com/62352288/235315735-afa4df88-74c4-48d8-a4c6-cf1645edd319.mov

Reply:
yeah this fix can work but how a user will know that more skills are there for a specific profile?

Reply:
Yeah, good point. For that we can add a `load more` feature as a pre-limiter and if `load more` is clicked, all the other skills be rendered.

Reply:
are you willing to work on it?

Reply:
Not sure for now, would you like to implement this feature, it's pretty easy to implement or else I can help you with a discussion on this as well.

Or Maybe you can raise an issue to implement this feature, for someone who's willing to contribute to it.

Let me know how do you want to proceed?

Good day :D

Reply:
No Problem I will look into it.

Reply:
@yashsehgal can you share the changes you made? I am having a issue.